code/global/Utils.cs

```diff
@@ -88,7 +103,7 @@ public static IEnumerable<Client> GetClientsSpectatingPlayer(TTTPlayer player)
 
         public static bool HasMinimumPlayers()
         {
-            return Client.All.Count >= Settings.ServerSettings.Instance.Round.MinPlayers;
+            return Client.All.Count - GetForcedSpectatingPlayers().Count >= Settings.ServerSettings.Instance.Round.MinPlayers;
```

Comment:
I'm not sure how much more performant, but we could just make a `GetNonForcedSpectatingPlayers()` call instead of doing a subtraction here.

Reply:
I agree. Will make this change.